site/app/templates/officeHoursQueue/QueueButtonBar.twig

```diff
@@ -16,18 +16,26 @@
   <button id="toggle_queue_message" class="btn btn-primary filter_btn" type="button" onclick="$('#queue-socket-message').show()">
     Send Message
   </button>
-  <button id="notification_btn" class="btn btn-default filter_btn" type="button" onclick="enableNotifications()">
-    Enable Notifications
-  </button>
   <a id="view-stats" class="btn btn-default filter_btn" href="{{base_url}}/stats">
     View Queue Stats
   </a>
-  <button class="btn btn-default filter_btn enable-audible" title="Enable audible alert when queue changes" onclick="toggleAudibleAlert()">
-    <i class="fas fa-bell" ></i>
-  </button>
-  <button class="btn btn-default filter_btn disable-audible" title="Disable audible alert when queue changes" onclick="toggleAudibleAlert()">
-    <i class="fas fa-bell-slash"></i>
-  </button>
+  <div class="super-switch-container" data-testid="notification-switch-container">
+    <div class="switch-container">
+      <span class="switch-label">Push Notifications</span>
+      <label class="switch" aria-label="Slider Switch for Push Notifications" data-testid="push-notification-switch">
+        <input type="checkbox" id="push-notification-switch" onclick="enableNotifications()">
```

Comment:
```suggestion
        <input type="checkbox" id="push-notification-switch" onclick="enableNotifications();">
```

Lets add semi-colon, it won't matter unless we add additional function but just stylistic suggestion